src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({
+  person: { name, age, sex, isMarried, partnerName },
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
```

Comment:
For better readability please place content on separate lines 
```suggestion
    <h2 className="Person__name">
      {`My name is ${name}`}
    </h2>
```